src/NHibernate/Dialect/Oracle8iDialect.cs

```diff
@@ -80,6 +94,15 @@ public Oracle8iDialect()
 			RegisterDefaultProperties();
 		}
 
+		/// <inheritdoc/>
+		public override void Configure(IDictionary<string, string> settings)
+		{
+			base.Configure(settings);
+
+			UseNPrefixedTypesForUnicode = PropertiesHelper.GetBoolean(Cfg.Environment.OracleUseNPrefixedTypesForUnicode, settings, false);
+			RegisterCharacterTypeMappings();
```

Comment:
Possible breaking change for those using hbm2ddl: now Unicode string types will be mapped by default to non `N` prefixed types, while NH 4 was mapping to `N` prefixed one.

Doing it the other way round would be a possible breaking change for queries instead, because they would start using `NVarchar2` for Unicode string parameters instead of `Varchar2` (NH 4 currently maps that to `Varchar2`).

Other possible breaking change, character types will not be registered if the dialect is not configured, which may happen if not getting it through `Dialect.GetDialect`.